safe/definitions/post_processors/__init__.py

```diff
@@ -41,6 +41,7 @@
      age_postprocessors +
      minimum_needs_post_processors +
      age_vulnerability_postprocessors +
-     disabled_vulnerability_postprocessors) + [
+     disabled_vulnerability_postprocessors +
```

Comment:
__[flake8]__
* [F405] 'disabled_vulnerability_postprocessors' may be undefined, or defined from star imports: minimum_needs_post_processors, population_post_processors, post_processor_functions, post_processor_inputs, post_processors

<!-- Comment by [SideCI](https://sideci.com) -->